openshift/template.yaml

```diff
@@ -233,6 +233,8 @@ objects:
                 value: ${FREE_ADDRESSES_IMAGE}
               - name: DHCP_LEASE_ALLOCATOR_IMAGE
                 value: ${DHCP_LEASE_ALLOCATOR_IMAGE}
+              - name: LOG_LEVEL
```

Comment:
I think there is a need to declare it as a param at the top of this file.
Right @ybettan ?

Reply:
Correct.
You are declaring a new env variable called `LOG_LEVEL` and initialise it with ${LOG_LEVEL} which doesn't exist anywhere. 
You need to declare LOG_LEVEL parameter *to the template* at the top of the file (see other parameters) and then making use of it will make cense.

Reply:
done